### DISCUSSION TITLE
Is this project alive?

### DISCUSSION THREAD

**Discussion by romuloalves** [dracula/dracula-ui#128] [General]
Hi guys!
I would like to know if this project is still maintained or died.

Thanks!

**Comment by activexray** [comment]
Well, considering it’s almost been a year since the last commit and there are still quite a few bugs….

**Comment by TCNOco** [comment]
Sad. I really was looking forward to seeing what changed... And nothing.
I do hope its development is resumed in the future.

**Comment by luxonauta** [comment]
Hi 👋🏻

Thank you all for your interest in the project; That means a lot to us. 💜

We are sorry for that; We are aware of the lack of updates. 😥

We plan to get back on track soon! 🚀 Thank you all for your patience.

**Comment by zenorocha** [comment]
The project is now free and open source for everybody to access and contribute.

Hopefully, we'll be able to get more updates and contributions from the community now that the project will have more exposure.
